README.nl.md

```diff
@@ -0,0 +1,66 @@
+# sway
+
+Sway is een [i3](https://i3wm.org/)-compatibele [Wayland](http://wayland.freedesktop.org/) compositor.
+Lees de [FAQ](https://github.com/swaywm/sway/wiki). Word lid van het [IRC
+kanaal](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway op
+irc.freenode.net).
+
+Als je de ontwikkeling van sway wilt ondersteunen, draag dan bij aan [SirCmpwn's
+Patreon-pagina](https://patreon.com/sircmpwn).
+
+## Releasehandtekeningen
+
+Releases worden gesingeerd met [B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A)
```

Comment:
Typo: gesigneerd?

Reply:
Someone else suggests 'ondertekend'

Reply:
Both sound alright to me, but note that my dutch is shaky (never learned to spell, never used a computer in dutch).

Reply:
ondertekend is correct dutch for this.